packages/react-button/src/components/MenuButton/MenuButton.types.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { BaseSlots, SlotProp, SlotProps } from '@fluentui/react-compose';
+import { IContextualMenuProps } from 'office-ui-fabric-react';
+import { ButtonProps, ButtonState, ButtonTokens } from '../Button/Button.types';
+
+export interface MenuButtonProps extends Omit<ButtonProps, 'icon' | 'iconPosition' | 'loader'> {
+  /**
+   * Menu that is displayed when the button is pressed.
+   */
+  // tslint:disable-next-line:no-any
+  menu?: SlotProp<Partial<IContextualMenuProps>>;
```

Comment:
hm why is `Partial` needed here?

Reply:
It was needed because using `mergeSlotProp` in `useMenuButton` on the `menu` slot required for me to send an `items` objects since that is required in the `IContextualMenuProps` interface. I've changed this to specify partial only on the `mergeSlotProp` call and not on the `menu` slot itself.